server/routes/forms/incentiveLevelCreateForm.ts

```diff
@@ -0,0 +1,17 @@
+import Form, { type BaseFormData } from './forms'
+
+export interface IncentiveLevelCreateData extends BaseFormData {
+  name: string
+  code: string
+}
+
+export default class IncentiveLevelCreateForm extends Form<IncentiveLevelCreateData> {
+  protected validate(): void {
+    if (!this.data.name || this.data.name.length < 1) {
+      this.addError('name', 'The level’s name is required')
+    }
+    if (!this.data.code || this.data.code.length < 1) {
+      this.addError('name', 'The level’s code is required')
```

Comment:
This should add the error to `code`, not `name` here right?

Reply:
fixed in a later commit!
https://github.com/ministryofjustice/hmpps-incentives-ui/pull/479/commits/f83f57118db79dd0efd72b6ac530100a5160cad9#diff-5e5411b99aae4cbc9cee6c93e72f93c8feb62d297bd4a5251f4aeb2a3abf3a50R18

Reply:
oh sorry, I was having a look commit-by-commit 👍